sdk/eventhub/azure-eventhub/out/azure/eventhub/aio/_consumer_client_async.pyi

```diff
@@ -0,0 +1,22 @@
+class EventHubConsumerClient(ClientBaseAsync):
+
+    #...
+
+    async def receive_batch(
+            self, on_event_batch: Callable[[PartitionContext, List[EventData]], Awaitable[None]],
+            *,
+            max_batch_size: int=...,
+            max_wait_time: float=...,
```

Comment:
I assume the unit of this is seconds, correct?

Reply:
yes